src/components/Upload.vue

```diff
@@ -28,8 +28,8 @@ v-card.fill-height(flat)
       v-alert(:value="true", dark, type="error") {{ errorMessage }}
         v-btn(v-if="!uploading", dark, outlined, @click="start") Resume upload
 
-    slot(name="files")
-      file-upload-list(v-model="files", v-bind="{ currentIndex, maxShow }")
+    slot(name="files", v-bind="{ files, setFiles, currentIndex, maxShow }")
+      file-upload-list(@input="setFiles", v-bind="{ value: files, currentIndex, maxShow }")
```

Comment:
This is actually a separate bugfix.  If files are deleted such that index >= files.length, the index should be changed.